source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -34,7 +34,7 @@ Before you start
    For the smoothest experience, we recommend that your computer has at least
    **16 GB of RAM, 2 CPUs, and at least 50 GB of Free disk space**.
 
-This tutorial is written for users of macOS, either with an Intel or an ARM
+This tutorial is written for users of macOS or Linux, either with an Intel or an ARM
```

Comment:
Has this been tested on Linux with an ARM CPU?
I know it works with AMD CPUs, but not sure about ARM.

Reply:
good point that it's a bit ambiguous. I re-arranged it so it's a bit more clear.